Documentation/docs.md

```diff
@@ -10,6 +10,7 @@ New etcd users and developers should get started by [downloading and building][d
 
 The easiest way to get started using etcd as a distributed key-value store for your applications is to [set up a local cluster][local_cluster].
 
+ - [Quick demo][demo]
```

Comment:
move this to getting started section? Add a sentence to say here is a quick demo?